src/Sulu/Bundle/WebsiteBundle/Resources/views/Sitemap/sitemap.xml.twig

```diff
@@ -4,7 +4,7 @@
 {% for site in sitemap if site.locale in locales %}
 {% if site.nodeType == 1 %}
 {% set siteLocale = site.locale %}
-{% set url = sitemap_url(site.url, site.locale) %}
+{% set url = sitemap_url(site.url, site.locale, site.webspaceKey) %}
```

Comment:
Is there a reason you pass the webspace here? Should work without it as well, which would be quite comfortable if the key changes